coffea/processor/__init__.py

```diff
@@ -5,7 +5,6 @@
 from .processor import ProcessorABC
 from .dataframe import (
     LazyDataFrame,
-    PreloadedDataFrame,
```

Comment:
Does this mean the spark executor only supports NanoEvents? Or just that it uses some other mechanism for generic root files?

Reply:
Spark executor works on the internals of spark which are pandas or arrow tables, which will always be convertible to awkward. There is little reason to support anything but nanoevents here.

Laurelin or whatever else is used as the input is completely decoupled.

Reply:
Ah right, so the base form is always reasonable